site/docs/tasks/using-opa.md

```diff
@@ -3,17 +3,39 @@
 
 When OPA is used for data governance, it is deployed as a stand-alone service.  Policies are defined in rego and uploaded to OPA.
 
+## Fybrik Default Policies
+
+Fybrik <strong><em>denys by default </em></strong>any request if no rule is triggered. This behavior can be changed to <strong><em>allow by default</em></strong> by creating the following rule and upload it to OPA using methods described in this page:
```

Comment:
It's **markdown** so there's no need for unnecessary html tags

Reply:
I followed https://www.markdownguide.org/basic-syntax/#bold-and-italic

Reply:
You followed the wrong column :)